### ISSUE TITLE
Add image builds for arm64

### ISSUE THREAD

**Issue by steveiliop56** [havenweb/haven#108]
Hello, awesome project! It would be awesome if the image supported arm64 too.

**Comment by Ogboji22** [comment]
I don't really understand what I am doing here I just found myself here I
really need help

**Comment by steveiliop56** [comment]
Oh it's super easy. Just use this:

```yaml
  build-image: 
     needs: create-tag 
     runs-on: ubuntu-latest 
     steps: 
       - name: Checkout 
         uses: actions/checkout@v4 
  
       - name: Set up QEMU 
         uses: docker/setup-qemu-action@v3 
  
       - name: Set up Docker Buildx 
         uses: docker/setup-buildx-action@v3 
  
       - name: Login to GitHub Container Registry 
         uses: docker/login-action@v3 
         with: 
           registry: ghcr.io 
           username: ${{ github.repository_owner }} 
           password: ${{ secrets.GITHUB_TOKEN }} 
  
       - name: Build and push image 
         uses: docker/build-push-action@v6 
         with: 
           context: . 
           platforms: linux/amd64, linux/arm64 
           push: true 
           tags: ghcr.io/${{ github.repository_owner }}/getashell:${{ needs.create-tag.outputs.tagname }}, ghcr.io/${{ github.repository_owner }}/getashell:latest
```

It will use docker buildx to build for both arm64 and amd64. If you like I can open a pull request.

**Comment by mawise** [comment]
Yes, please do send a PR!

**Comment by steveiliop56** [comment]
Alright give me a moment!

**Comment by steveiliop56** [comment]
Here you go #110

**Comment by mawise** [comment]
Fixed in #110 
Thank you @steveiliop56 !
